MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
Please add your name. Kindly be your Copyright

Reply:
- [ ] Here as well it could be good to say why the student needs to be his Copyright 😉